### DISCUSSION TITLE
Proposal for Akash Network Workshop in Indore and Bangalore

### DISCUSSION THREAD

**Discussion by hardiktheprogrammer** [akash-network/community#589] [Governance Proposals]
### Workshop Description

We propose a technical workshop focused on the Akash Network, tailored for developers. The primary goal is to introduce the Akash Network and the Secret Network, provide a hands-on guide for setting up an Akash provider, and explore building a cloud provider. This workshop aims to empower developers with the knowledge and skills to leverage decentralized cloud computing.

#### Locations and Logistics

**1. Bangalore:**
   - **Venue:** TBA
   - **Date:**   23 June
   - **Time:** 9:00 AM - 2:00 PM

**2. Indore:**
   - **Venue:** IIT Indore
   - **Date:**   19 June
   - **Time:** 9:00 AM - 2:00 PM

**Logistics:**
   - Access to microphones, projectors, standees, and banners
   - Seating arrangement for up to 70-80 participants
   - Refreshments and lunch for participants
   - Workshop swags - akash sticker,merch

#### Event Formation

**1. Introduction to Akash Network (30-40 mins):**
   - Overview of Akash Network
   - Key features and advantages
   - Integration with Secret Network

**2. Setting Up Akash Provider (1 hour):**
   - Hardware and software requirements
   - Installation process
   - Configuration and deployment
   - Q&A session

**3. Building a Cloud Provider ( 40-60 min):**
   - Tools and frameworks required
   - Hands-on building session
   - Best practices and tips

**4. Interactive Session (1 hour):**
   - Group activities and challenges
   - Networking opportunities

#### Promotion Plans

1. **Social Media Campaign:**
   - Leverage platforms like Twitter, LinkedIn, and Telegram
   - Promote through tweets and local community groups

3. **Local Community Outreach:**
   - Collaborate with university students and developer clubs such as blockchain clubs in university 

#### Community Engagement

- Create a dedicated workshop channel on the Akash telegram channel
- Encourage participants to share their progress and queries
- Post-workshop support through dedicated mentors and resources

#### Timeline

- 2 Weeks Before the Event Finalize venues ,dates and Launch promotional campaign
- Engage participants and collect feedback through telegram or what’sapp group 

#### Conclusion

Hosting a 4-5 hour workshop on the Akash Network in Indore and Bangalore will significantly enhance the visibility of Akash in these key tech hubs. By focusing on developers, we aim to build a strong community of users and contributors to the Akash Network. With a comprehensive promotional plan and community engagement strategy, this workshop will be a stepping stone towards greater adoption and innovation within the decentralized cloud computing space.

**Comment by brewsterdrinkwater** [comment]
Hello @hardiktheprogrammer 

Thank you for putting this together. Are you interested in finding someone from the Akash community to give this talk, or are you volunteering yourself to give the 4-5 workshop? 

Are you going to be collecting leads from the participants?

The idea of working with university students is something that is high priority for many folks in the Akash community. 

Is there an ask for this proposal?
